lol_alloc/src/free_list_allocator.rs

```diff
@@ -173,14 +173,13 @@ fn full_size(layout: Layout) -> usize {
 // From https://github.com/wackywendell/basicalloc/blob/0ad35d6308f70996f5a29b75381917f4cbfd9aef/src/allocators.rs
 // Round up value to the nearest multiple of increment
 fn round_up(value: usize, increment: usize) -> usize {
-    if value == 0 {
-        return 0;
-    }
-    increment * ((value - 1) / increment + 1)
+    debug_assert!(increment.is_power_of_two());
+    (value + (increment - 1)) & increment.wrapping_neg()
 }
 
 fn multiple_below(value: usize, increment: usize) -> usize {
```

Comment:
This was missing a comment noting that multiples of increment are left unchanged. I know that my issue from before this PR, bug if adding a doc comment here, that would help.

Reply:
Done.